### PR TITLE
chore: Merge master into v12.5.0 (following v12.4.2)

### PR DESCRIPTION
**Pull request by hjetpoluru**
## **Description**

PR to merge hotfix changes of v12.4.2 into v12.5.0

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/27938?quickstart=1)

### CODE REVIEW

**Review by benjisclowder — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -4249,9 +4249,6 @@
   "receive": {
     "message": "Receive"
   },
```
Comment (danjm) [inline app/_locales/en/messages.json:4251]: This code is present on develop, and so it probably should not be deleted: https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L4323-L4325

File: app/_locales/en_GB/messages.json [hunk 2]
```diff
@@ -4180,7 +4180,7 @@
   "receive": {
     "message": "Receive"
   },
```
Comment (danjm) [inline app/_locales/en_GB/messages.json:4182]: On develop, this is "recipientAddressPlaceholder": https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en_GB/messages.json#L4143-L4145

Comment (hjetpoluru) [reply app/_locales/en_GB/messages.json:4182]: I agree with you that the develop shows as posted by you.

But in v12.4.2 has this changes - https://github.com/MetaMask/metamask-extension/pull/27879/files#diff-7509d7529ea1cbc57ef79713676f7d124d9d5a446053f94836876c4cd3f3ccd6

Hence I took the v12.4.2. Now my current question is how did this code get to hotfix only?

File: app/_locales/en/messages.json (hunk 1 above)
Comment (hjetpoluru) [reply app/_locales/en/messages.json:4251]: I agree with you that the develop shows as posted by you.

But in v12.4.2 has this changes - https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L4323-L4325

Hence I took the v12.4.2. Now my current question is how did this code get to hotfix only?

Comment (hjetpoluru) [reply app/_locales/en/messages.json]: ok thanks @danjm, I have address the conflict with the instruction you have provided.

File: ui/ducks/send/send.js [hunk 3]
```diff
@@ -109,6 +109,7 @@ import {
   isBurnAddress,
   isPossibleAddress,
   isValidHexAddress,
+  isPossibleAddress,
```
Comment (danjm) [inline ui/ducks/send/send.js:112]: This change probably needs to be removed for lint to pass

File: ui/ducks/send/send.test.js [hunk 4]
```diff
@@ -768,7 +768,7 @@ describe('Send Slice', () => {
       it('should not error with an invalid address error when user input is not a valid hex string', () => {
         const tokenAssetTypeState = {
           ...INITIAL_SEND_STATE_FOR_EXISTING_DRAFT,
-          recipientInput: '0xValidateError',
+          recipientInput: '0xAAAA6BF26964aF9D7eEd9e03E53415D37aA96045',
         };
```
Comment (danjm) [inline ui/ducks/send/send.test.js:772]: This change probably needs to be removed for tests to pass

Comment (hjetpoluru) [reply ui/ducks/send/send.test.js:772]: @danjm, Just to confirm - is this test not needed and currently not present in the develop branch and should it be removed?

Comment (hjetpoluru) [reply ui/ducks/send/send.test.js]: ok done.

File: ui/ducks/send/send.js (hunk 3 above)
Comment (hjetpoluru) [reply ui/ducks/send/send.js]: Sure done.

**Review by danjm — APPROVED** [review]

**Pull request merged**